contracts/storage/ParametersStorage.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+
+
```

Comment:
the one thing we should figure out is - how do we add parameters to this contract over time?

Reply:
@u-hubar a hub approach + storage and implementation separation come to mind.